vng_api_common/validators.py

```diff
@@ -240,15 +240,22 @@ def __call__(self, informatieobject: str, serializer):
         # dynamic so that it can be mocked in tests easily
         client = get_client(informatieobject)
 
+        if not client:
+            raise ValidationError(
+                _("Geen service geconfigureerd voor %s") % informatieobject
+            )
+
         try:
-            oios = client.list(
+            response = client.get(
                 "objectinformatieobject",
-                query_params={
+                params={
                     "informatieobject": informatieobject,
                     "object": object_url,
                 },
             )
-        except requests.HTTPError as exc:
+
+            oios = to_internal_data(response)
+        except (requests.RequestException, RuntimeError) as exc:
```

Comment:
Why don't you catch ClientError here? Or is it caught with `requests.RequestException` ?

Reply:
It seems that `ClientError` is based on `RuntimeError`, but I'll change it to explicitly catch `ClientError` instead :+1:

Reply:
Upon checking, this validator isn't used anywhere in Open Zaak (which uses its own `CreateRemoteRelationValidator`), so I'll remove it